Lambda-School-Labs/CS10-quizzer
# Abbott/front end standard eslint

# Description

Updates current front-end client code to follow [Standard JS](https://standardjs.com/index.html) styling conventions (should have no effect on PR's for server-side code). Also updates README with badge for Standard JS usage.

Flagged as 'bug fix' as this "fixes" our lack of established JS styling.

## Type of change


# How Has This Been Tested?

* Install relevant packages with `yarn`.
* Run `yarn start` to begin development server.
* No errors or warnings appear in console, landing page loads normally.

# Checklist:
